src/App.vue

```diff
@@ -29,12 +36,16 @@ export default {
     ...mapState([
       'selectedFont',
       'fonts',
+      'showDarkMode'
     ])
   },
   watch: {
     selectedFont(value) {
       updateSelectedFont(value)
     },
+    showDarkMode(value){
+      value ? document.body.classList.add("dark-mode") : document.body.classList.remove("dark-mode");
```

Comment:
Acho que invés de colocar essa classe no body vc pode colocar no `<div id="app">` mesmo já que todos os outros components da aplicação tão dentro dele, dai to pode fazer o app ser do tamanho da tela com css saca.